modules/tspec/assert.go

```diff
@@ -0,0 +1,38 @@
+package tspec
+
+import (
+	"fmt"
+
+	"github.com/stretchr/testify/assert"
+)
+
+// assertExpectedAndActual is a helper function to allow the step function to call
+// assertion functions where you want to compare an expected and an actual value.
+func assertExpectedAndActual(a expectedAndActualAssertion, expected, actual interface{}, msgAndArgs ...interface{}) error {
+	var t asserter
+	a(&t, expected, actual, msgAndArgs...)
```

Comment:
NIT: try to avoid one-letter variable names like `a`. Even though it's defined just 2 lines above, but the time you read this line, it's completely opaque what `a` is supposed to be.